docs/serverless/08-04-trigger-function-with-event.md

```diff
@@ -105,17 +105,46 @@ To test if the Trigger CR is properly connected to the Function:
     }
     ```
 
-2. Send an event manually to trigger the Function:
+2.  Send an event manually to trigger the function. The first example shows the implementation introduced with Kyma 1.11 release, where a [CloudEvent](https://github.com/cloudevents/spec/blob/v1.0/spec.md) is sent directly to the Event Mesh. In the second example, an event also reaches the Event Mesh, but before that it is modified by the compatibility layer to the format compliant with the CloudEvents specification. This solution ensures compatibility if your events follow a format other than CloudEvents, or you used the Event Bus available before 1.11.
```

Comment:
- Kyma 1.11 release -> Kyma the 1.11 release
- but before that it is modified by -> but it is first modified by
- or you used the Event Bus -> or you use the Event Bus

Reply:
just `the Kyma 1.11 release`?